python_modules/dagster/dagster/_core/definitions/load_assets_from_modules.py

```diff
@@ -313,3 +308,21 @@ def asset2(asset1):
             )
         )
     return result_assets
+
+
+def group_assets(
```

Comment:
`with_group` might be more consistent with other functions and methods that do similar things